packages/discord-bot/src/index.ts

```diff
@@ -39,7 +39,14 @@ discordClient.on('interactionCreate', async (interaction) => {
   const command = discordClient.commands.get(interaction.commandName);
   if (!command) return;
   try {
-    await command.execute(interaction);
+    if (
+      interaction.commandName === 'praise' ||
+      interaction.commandName === 'forward'
+    ) {
+      await command.execute(interaction, discordClient);
+    } else {
+      await command.execute(interaction);
+    }
```

Comment:
I think you can just drop this conditional entirely -- the functions will just ignore any additional parameters they're not expecting.

i.e.
```
  try {
    await command.execute(interaction, discordClient);
  } catch (error) {
  ...
  ```